python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -642,6 +649,7 @@ cdef class ParquetWriter:
             if use_dictionary
             else cudf_io_types.dictionary_policy.NEVER
         )
+        self.write_arrow_schema = store_schema
```

Comment:
Should we change the internal arg name write_arrow_schema to store_schema (used in pyarrow) or let it be?

Reply:
Answered in cuIO standup (Wed 5/29): let it be like this